libraries/stdlib/common/src/generated/_Arrays.kt

```diff
@@ -6338,6 +6338,51 @@ public expect fun BooleanArray.copyOfRange(fromIndex: Int, toIndex: Int): Boolea
  */
 public expect fun CharArray.copyOfRange(fromIndex: Int, toIndex: Int): CharArray
 
+/**
+ * Fills original array with the provided value.
+ */
+public expect fun <T> Array<T>.fill(element: T, fromIndex: Int = 0, toIndex: Int = size): Unit
```

Comment:
Isn't `Array<in T>` enough here?

Reply:
This is the original signature of the function. The PR just moves it to common.

If we want a signature change I would prefer to do it separately.

Reply:
@Prototik, the current signature won't benefit from changing `Array<T>` to `Array<in T>`.